common/thorhelper/csvsplitter.cpp

```diff
@@ -73,7 +73,7 @@ void CSVSplitter::addTerminator(const char * text)
 
 void CSVSplitter::addEscape(const char * text)
 {
-    matcher.addEntry(text, ESCAPE);
+    matcher.queryAddEntry((size32_t)strlen(text), text, ESCAPE);
```

Comment:
@richardkchapman as you wish.

BTW: I think this is a bit over-sophisticated solution (add extra function call) to do a simple thing: if 'no escape' then do nothing.
Based on:
```
bool StringMatcher::queryAddEntry(unsigned len, const char * text, unsigned action)
{
    if (len == 0)
        return false;
```